scripts/label_for_url.py

```diff
@@ -298,6 +298,26 @@ def determine_label(input_url: str) -> str:
         )
         page_id = f"/{m.group('pageId')}" if m.group("pageId") else ""
         return f"{space}{title}{anchor}" or f"{m.group('host')}{page_id}"
+    elif m := re.search(
+        r"^https://(?P<org>.*atlassian.*)"
+        r"/wiki/spaces/(?P<space>[^/]+)"
+        r"(?:/pages/(?P<pageId>.*/(?P<title>[^?#]+))?)?"
+        r"\??(?P<args>.*)",
+        input_url,
+    ):
+        space = m.group("space") if m.group("space") else ""
+        title = f"/{prettify(m.group('title'))}" if m.group("title") else ""
+        args = m.group("args").split("&")  # ["focusedCommentId=43245234", "test=abc"]
+
+        args = args[0].split("=") if args[0].startswith("focusedCommentId") else args
```

Comment:
:broom:  We repeat `if _.startswith("focusedCommentId")` often enough that an actual if-then-else seems appropriate, and is probably clearer.